app/routes/speak.tsx

```diff
@@ -35,9 +35,14 @@ export default function About() {
 			<UnorderedList>
 				<li>
 					Send us your slides 24 hours in advance, so we can check them for
-					accessibility
+					accessibility and/or dry run your talk with us the week before
```

Comment:
I believe we're going with dry runs as non-optional:

```tsx
<li>
  Present a dry run of your talk the week before, so we can check it 
  for accessibility issues and make sure it runs the expected duration.
</li>

Reply:
Done!